cfme/tests/infrastructure/test_snapshot.py

```diff
@@ -29,7 +29,7 @@
 ]
 
 
-@pytest.fixture(scope="module")
+@pytest.fixture(scope="function")
 def domain(request, appliance):
```

Comment:
Suggestion:
As per me we have one global fixture for `domain`
`https://github.com/ManageIQ/integration_tests/blob/master/cfme/fixtures/automate.py#L6`

Please have look it will helpful to remove this stuff. let me know your view on this.

Reply:
thanks! I ran a test `test_create_snapshot_via_ae` with a global fixture and it works